integrations/node/workload-executor.js

```diff
@@ -87,7 +91,7 @@ function runOperation(op) {
     throw new Error(`Unsupported operation: ${op.name}`);
   }
   const operation = operations.get(op.name);
-  return operation(object, op.arguments);
+  return operation(object, JSON.parse(JSON.stringify(op.arguments)));
```

Comment:
Let's wrap this in a `try/catch` to make it very obvious to future maintainers why this fails if it does